### DISCUSSION TITLE
Add support to change external display brightness?

### DISCUSSION THREAD

**Discussion by Maccer1** [elementary/wingpanel-indicator-power#228] [Ideas]
With the magic of DCC/CI it is possible to control external monitors from the command line.

I've explained [here](https://github.com/elementary/switchboard-plug-power/issues/173) how it is possible to change brightness. Since this feature seems quite the task I just wanted to share the idea with you. I believe this feature is not well known so I just wanted to create some awareness.

**Comment by Maccer1** [comment]
In short: `sudo ddcutil setvcp 10 XX` where XX is the brightness value. Try between 0-100.

**Comment by danirabbit** [comment]
This is a duplicate of this issue report: https://github.com/elementary/wingpanel-indicator-power/issues/41
